data/schema.txt

```diff
@@ -1131,6 +1131,40 @@ CREATE TABLE `ref_snp_info` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
+-- Table structure for table `rna_seqc`
+--
+
+DROP TABLE IF EXISTS `rna_seqc`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `rna_seqc` (
+  `id_rna_seqc` bigint(20) unsigned NOT NULL AUTO_INCREMENT COMMENT 'Auto-generated primary key',
+  `id_seq_composition` bigint(20) unsigned NOT NULL COMMENT 'A foreign key referencing the id_seq_composition column of the seq_composition table',
+  `path` varchar(256) DEFAULT NULL,
```

Comment:
I am not sure that this field is going to be filled by this check. This is a historic feature, which we probably can discontinue.